### ISSUE TITLE
Question about dir `replacement`

### ISSUE THREAD

**Issue by Zander-J** [cvdfoundation/kinetics-dataset#45]
I have downloaded the data and extracted the targz files using .sh scripts. Then how can I use `replacement` data? Should I move all the files in `replacement` to any other dir?

**Comment by rendicahya** [comment]
@A91A981E No, you should not move the files manually. Instead, run `arrange_by_classes.py` and it will do the work. However, I found a small error and I have submitted a PR. Please take a look: https://github.com/cvdfoundation/kinetics-dataset/pull/46.

**Comment by Zander-J** [comment]
Oh! I get it. Thanks for telling me this and your PR!
